### ISSUE TITLE
Can RFAA do protein-protein interaction prediction？

### ISSUE THREAD

**Issue by hcpxyz** [baker-laboratory/RoseTTAFold-All-Atom#156]
I didn't know if this worked, but I put a few proteins in protein.yaml and see if I could get the output. The last line shows Killed. Does it mean that RFAA can't do PPI prediction or the format in protein.yaml is wrong?
![screenshot_2024-12-17_11-50-59](https://github.com/user-attachments/assets/6418075a-446b-4b3a-a52d-4afa4034e017)

**Comment by hcpxyz** [comment]
I tried 4 proteins, and I got the pdb file. It looks fine in pymol.
![image](https://github.com/user-attachments/assets/34b0531a-dd7e-4bc0-be5c-ccdf4ab4742b)
So I thought there might be something wrong with the 5th protein, and I ran the 5th protein alone. I also got the pdb file of the 5th protein. So I was wondering if RFAA might not do PPI prediction when 5 proteins were involved, and the max number of proteins was four.
